assets/scss/brandings.scss

```diff
@@ -55,6 +57,7 @@
 		#ccfw-page-banner {
 			.ccfw-banner__button:not(.ccfw-banner__button--close) {
 				background-color: branded(cookie-button-bg);
+				background-color: var(--cookie-button-background);
```

Comment:
The non-IE11 method allows us to reference the new colour variable in the current CSS structure, keeping the new code in the current SCSS structure.